apps/site-landing/src/components/RangeFieldAndScale/index.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import { useState } from 'react';
+import styles from './styles.module.less';
+import { FormattedMessage, FormattedNumber, MessageDescriptor } from 'react-intl';
+import type { NumberFormatOptionsStyle } from '@formatjs/ecma402-abstract';
+
+import { Range, getTrackBackground } from "react-range";
+import { LessVars } from "@thecointech/site-semantic-theme/variables";
+
+export type Props = {
+ scaleType?: NumberFormatOptionsStyle;
+ currency?: "CAD",
+ label: MessageDescriptor,
+ initial?: number;
+ maximum: number;
+ minimum?: number;
+ step?: number;
+ onChange: (value: number) => void;
+}
+
+export const RangeFieldAndScale = (props: Props) => {
+
+    const [value, setValue] = useState(props.initial ?? 0);
+
+    const minimum = props.minimum ?? 0;
+    const maximum = props.maximum;
+    const step = props.step ?? 1;
+
+    const onChange = ([value]: number[]) => {
+      setValue(value);
+      props.onChange(value);
+    }
+
+    return (
+      <div className={styles.container}>
+        <div className={styles.label}>
+          <FormattedMessage {...props.label} tagName="span" />
+          <FormattedNumber
+            value={value}
+            currency={props.currency}
+            currencyDisplay={"narrowSymbol"}
+            style={props.scaleType}
+            minimumFractionDigits={0}
+            maximumFractionDigits={0}
+           />
+        </div>
+        <Range
```

Comment:
I couldn't get the in-build one to color the track non-uniformly, so installed a new version to do it.